geanypg/src/key_selection_dialog.c

```diff
@@ -156,8 +155,8 @@ int geanypg_encrypt_selection_dialog(encrypt_data * ed, gpgme_key_t ** selected,
 
 
     /* add ok and cancel buttons */
-    gtk_dialog_add_button(GTK_DIALOG(dialog), GTK_STOCK_OK, GTK_RESPONSE_OK);
-    gtk_dialog_add_button(GTK_DIALOG(dialog), GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL);
```

Comment:
BTW, being deprecated only means they'll probably disappear in GTK 4.0, nothing else.